src/main.py

```diff
@@ -43,7 +43,7 @@ def main() -> None:
     """Main entry point for running approaches in environments.
     """
     if not os.path.exists("results/"):
-        os.mkdir("results/")
+        os.mkdir("results/") # pragma: no cover
```

Comment:
why is this needed?

Reply:
without this, we only get 99% coverage

Reply:
hmm, but that wasn't true before, so what changed?

Reply:
good point - I'm not really sure; I just tested this again to verify and pytest complains without it

Reply:
ah, I think what happens is that `results` doesn't exist on github checks, but it exists for us locally. so I bet if you removed the pragma: no cover here, the checks would pass on github, even though they don't locally. so yeah we can keep this, and @ronuchit said he'll also add it into a PR that he wants to merge soon.